pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -316,6 +308,12 @@ cdef dict c_DEPR_ABBREVS = {
     "S": "s",
 }
 
+PERIOD_AND_OFFSET_ALIASES = {
+    "W",
+    "min",
+}
```

Comment:
is it really only these two? should 's' and others not be in here too?

Reply:
this dictionary is specifically for frequencies which:
- are deprecated for wrong casing (but not removed)
- are the same for offsets and period

As a follow up we will add to this dictionary frequencies: `'D', 'C', 'B'.`

I didn't add to this dictionary frequencies such as: `'s', 'ms', 'us', 'ns'` because these frequencies should to be removed (in this PR)